core/src/com/unciv/models/translations/TranslationFileWriter.kt

```diff
@@ -484,18 +484,16 @@ object TranslationFileWriter {
      *  @param [translations] A [Translations] instance with all languages loaded.
      *  @return Success or error message.
      */
-    private fun writeTranslatedFastlaneFiles(translations: Translations): String {
-        try {
+    private fun writeTranslatedFastlaneFiles(translations: Translations): String = try {
             writeFastlaneFiles(shortDescriptionFile, translations[shortDescriptionKey], false)
             writeFastlaneFiles(fullDescriptionFile, translations[fullDescriptionKey], true)
             updateFastlaneChangelog()
 
-            return "Fastlane files are generated successfully."
+            "Fastlane files are generated successfully."
         } catch (ex: Throwable) {
             ex.printStackTrace()
-            return ex.localizedMessage ?: ex.javaClass.simpleName
+            ex.localizedMessage ?: ex.javaClass.simpleName
         }
-    }
```

Comment:
Not sure I agree with this change; using equality for a function to me indicates the function is very simple, at most 2 lines. Here it is just used to remove the word `return` twice, which only lowers readability imo.

Reply:
I was hesitated to do that whether it improves or decreases readability. Will revert.